package.json

```diff
@@ -98,6 +98,18 @@
               ""
             ],
             [
+              "1597",
+              "[@nikgraf](https://github.com/nikgraf)/[**github**](https://github.com/nikgraf/belle)",
```

Comment:
that will result in [@nikgraf](https://github.com/nikgraf)/[**github**](https://github.com/nikgraf/belle) , you might want to rename it to [@nikgraf](https://github.com/nikgraf)/[**belle**](https://github.com/nikgraf/belle) :)